Readme.md

```diff
@@ -273,6 +273,10 @@ Implements the `Date` object but using the clock to provide the correct time.
 
 Implements the `now` method of the [`Performance`](https://developer.mozilla.org/en-US/docs/Web/API/Performance/now) object but using the clock to provide the correct time. Only available in environments that support the Performance object (browsers mostly).
 
+### `lolex.lolexFromGlobal`
+
+In order to support creating clocks based on fake environments (such as JSDOM), Lolex exports a factory method which takes single argument `global`, which it inspects to figure out what to mock and what features to support. When invoking this function with a global, you will get back an object with `timers`, `createClock` and `install` - same as the regular Lolex exports only based on the apssed in global instead if the global environment.
```

Comment:
On the last line, "apssed" should presumably be "passed" and the "if" replaced with "of".

Could you perhaps refer to "separate environments" instead of "fake environments"? The latter description is not particularly accurate for jsdom.

Reply:
Good call! and thanks for the typo callouts 🙂